### PR TITLE
fixes RHAORDOC-1390

### PR DESCRIPTION
**Pull request by inoxx03**
updated references to latest Vertx version and Vertx Maven plugin

@huntley FYI^^

### CODE REVIEW

File: docs/topics/templates/document-attributes.adoc [hunk 1]
```diff
@@ -104,8 +104,8 @@
 :SpringBootMvnPluginVersion: 1.5.14.RELEASE
 
 // used in the BOM file example and in the Vert.x Runtime developer Guide
-:VertXVersion: 3.5.1.redhat-004
-:VertXMvnPluginVersion: 1.0.15
+:VertXVersion: 3.5.3.redhat-00001
```
Comment (rhoads-zach) [inline docs/topics/templates/document-attributes.adoc]: @inoxx03 is `00001` a typo?

Comment (huntley) [reply docs/topics/templates/document-attributes.adoc]: Do not believe so, that is the version in the BOM though I know it is different from previous releases.

Comment (inoxx03) [reply docs/topics/templates/document-attributes.adoc]: @rhoads-zach Can confirm, not a typo. I've checked the version against the BOM file and it checks out. Not sure why the string changed, could be due to tooling updates on the Vertx productization side.

Comment (rhoads-zach) [reply docs/topics/templates/document-attributes.adoc]: thank you @huntley and @inoxx03

**Comment by rhoads-zach** [comment]
@inoxx03 @huntley can i merge this?

File: docs/topics/templates/document-attributes.adoc (hunk 1 above)
Comment (Ladicek) [reply docs/topics/templates/document-attributes.adoc]: This is actually a new productization convention. All products will have the numeric suffix in a 5-digit form.

**Pull request merged**